From bd4715ce1bc4964e59c5352cf61ddf8bdef1a95a Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Tue, 11 Aug 2015 15:16:00 +0800
Subject: [PATCH 0168/5242] MLK-11345 ARM: dts: imx7d-12x12-lpddr3-arm2:
 correct lcdif pinctrl settings

commit  1ccde3ccdc95696fd612de5dd97dabbc631d4820 from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct the lcdif pinctrl settings to make it work
correctly.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |   56 ++++++++++++-------------
 1 file changed, 28 insertions(+), 28 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index c12aac2..c04d436 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -317,39 +317,39 @@
 
 		pinctrl_lcdif_dat: lcdifdatgrp {
 			fsl,pins = <
-				MX7D_PAD_EPDC_DATA00__LCD_DATA0  0x4001b0b0
-				MX7D_PAD_EPDC_DATA01__LCD_DATA1  0x4001b0b0
-				MX7D_PAD_EPDC_DATA02__LCD_DATA2  0x4001b0b0
-				MX7D_PAD_EPDC_DATA03__LCD_DATA3  0x4001b0b0
-				MX7D_PAD_EPDC_DATA04__LCD_DATA4  0x4001b0b0
-				MX7D_PAD_EPDC_DATA05__LCD_DATA5  0x4001b0b0
-				MX7D_PAD_EPDC_DATA06__LCD_DATA6  0x4001b0b0
-				MX7D_PAD_EPDC_DATA07__LCD_DATA7  0x4001b0b0
-				MX7D_PAD_EPDC_DATA08__LCD_DATA8  0x4001b0b0
-				MX7D_PAD_EPDC_DATA09__LCD_DATA9  0x4001b0b0
-				MX7D_PAD_EPDC_DATA10__LCD_DATA10 0x4001b0b0
-				MX7D_PAD_EPDC_DATA11__LCD_DATA11 0x4001b0b0
-				MX7D_PAD_EPDC_DATA12__LCD_DATA12 0x4001b0b0
-				MX7D_PAD_EPDC_DATA13__LCD_DATA13 0x4001b0b0
-				MX7D_PAD_EPDC_DATA14__LCD_DATA14 0x4001b0b0
-				MX7D_PAD_EPDC_DATA15__LCD_DATA15 0x4001b0b0
-				MX7D_PAD_EPDC_SDLE__LCD_DATA16   0x4001b0b0
-				MX7D_PAD_EPDC_SDOE__LCD_DATA17   0x4001b0b0
-				MX7D_PAD_EPDC_SDSHR__LCD_DATA18  0x4001b0b0
-				MX7D_PAD_EPDC_SDCE0__LCD_DATA19  0x4001b0b0
-				MX7D_PAD_EPDC_SDCE1__LCD_DATA20  0x4001b0b0
-				MX7D_PAD_EPDC_SDCE2__LCD_DATA21  0x4001b0b0
-				MX7D_PAD_EPDC_SDCE3__LCD_DATA22  0x4001b0b0
-				MX7D_PAD_EPDC_GDCLK__LCD_DATA23  0x4001b0b0
+				MX7D_PAD_LCD_DATA00__LCD_DATA0  0x79
+				MX7D_PAD_LCD_DATA01__LCD_DATA1  0x79
+				MX7D_PAD_LCD_DATA02__LCD_DATA2  0x79
+				MX7D_PAD_LCD_DATA03__LCD_DATA3  0x79
+				MX7D_PAD_LCD_DATA04__LCD_DATA4  0x79
+				MX7D_PAD_LCD_DATA05__LCD_DATA5  0x79
+				MX7D_PAD_LCD_DATA06__LCD_DATA6  0x79
+				MX7D_PAD_LCD_DATA07__LCD_DATA7  0x79
+				MX7D_PAD_LCD_DATA08__LCD_DATA8  0x79
+				MX7D_PAD_LCD_DATA09__LCD_DATA9  0x79
+				MX7D_PAD_LCD_DATA10__LCD_DATA10 0x79
+				MX7D_PAD_LCD_DATA11__LCD_DATA11 0x79
+				MX7D_PAD_LCD_DATA12__LCD_DATA12 0x79
+				MX7D_PAD_LCD_DATA13__LCD_DATA13 0x79
+				MX7D_PAD_LCD_DATA14__LCD_DATA14 0x79
+				MX7D_PAD_LCD_DATA15__LCD_DATA15 0x79
+				MX7D_PAD_LCD_DATA16__LCD_DATA16 0x79
+				MX7D_PAD_LCD_DATA17__LCD_DATA17 0x79
+				MX7D_PAD_LCD_DATA18__LCD_DATA18 0x79
+				MX7D_PAD_LCD_DATA19__LCD_DATA19 0x79
+				MX7D_PAD_LCD_DATA20__LCD_DATA20 0x79
+				MX7D_PAD_LCD_DATA21__LCD_DATA21 0x79
+				MX7D_PAD_LCD_DATA22__LCD_DATA22 0x79
+				MX7D_PAD_LCD_DATA23__LCD_DATA23 0x79
 			>;
 		};
 
 		pinctrl_lcdif_ctrl: lcdifctrlgrp {
 			fsl,pins = <
-				MX7D_PAD_EPDC_SDCLK__LCD_CLK 		0x4001b0b0
-				MX7D_PAD_EPDC_BDR1__LCD_ENABLE 		0x4001b0b0
-				MX7D_PAD_EPDC_PWR_STAT__LCD_VSYNC 	0x4001b0b0
-				MX7D_PAD_EPDC_PWR_COM__LCD_HSYNC 	0x4001b0b0
+				MX7D_PAD_LCD_CLK__LCD_CLK       0x79
+				MX7D_PAD_LCD_ENABLE__LCD_ENABLE 0x79
+				MX7D_PAD_LCD_VSYNC__LCD_VSYNC   0x79
+				MX7D_PAD_LCD_HSYNC__LCD_HSYNC   0x79
 			>;
 		};
 
-- 
1.7.9.5

